dockstore-webservice/src/main/java/io/swagger/model/ToolV1.java

```diff
@@ -0,0 +1,38 @@
+/*
+ *    Copyright 2017 OICR
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+package io.swagger.model;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonNaming(PropertyNamingStrategy.KebabCaseStrategy.class)
+public class ToolV1 extends ToolPresenter {
+    public ToolV1(Tool tool) {
+        this.tool = tool;
+    }
+
+    @JsonProperty("versions")
+    public List<ToolVersionV1> getVersionsV1() {
+        List<ToolVersion> toolVersions = this.tool.getVersions();
+        return toolVersions.parallelStream().map(toolVersion -> new ToolVersionV1(toolVersion)).collect(Collectors.toList());
```

Comment:
BTW, I agree with @coverbeck on this, parallelStream is probably more complex than needed here. So which to a normal stream when templating this with the V2 version, thanks!